cmd/logstash/README.md

```diff
@@ -43,7 +45,6 @@ git checkout tags/v7.6.2
 export LOGSTASH_PATH=`pwd`
 export GEM_PATH=$LOGSTASH_PATH/vendor/bundle/jruby/2.5.0
 export GEM_HOME=$LOGSTASH_PATH/vendor/bundle/jruby/2.5.0
-./gradlew assemble
```

Comment:
That what did the trick for me. I needed the jar to run the plugin. It’s also documented.

Reply:
The jar is required if we are doing Java development. But Ruby based development should not require this. Anyways I will add it back.

Reply:
The plugin could not run for me without this